README.md

```diff
@@ -32,10 +35,10 @@ registration.agree_terms
 # We need to prove that we control the domain using one of the challenges method.
 authorization = client.authorize(domain: 'example.org')
 
-# For now the only challenge method supprted by the client is http-01.
+# For now the only challenge method supported by this client is http-01.
```

Comment:
I did not dig into the code, can you tell me what other method are supported? Would it make sense to update the readme with example from other methods?